scripts/fair-eva.py

```diff
@@ -130,6 +130,17 @@ def main():
     args = get_input_args()
     url = args.api_endpoint
 
+    if args.repository == None:
+        response = requests.get(
+            "http://localhost:9090/v1.0/endpoints?plugin=" + args.plugin
+        )
+        if response.text == "Input plugin not found":
```

Comment:
better to rely on the response code (see the comment above about `endpoints()` method returning HTTP code)

Reply:
Now it uses status codes